charts/seed-bootstrap/prometheus-rules/recording-rules.rules.yaml

```diff
@@ -27,6 +42,21 @@ groups:
   - record: seed:container_network_transmit_bytes_total:sum
     expr: sum(seed:container_network_transmit_bytes_total:sum_by_pod)
 
+  # Recording rules for the sum of the entire seed requests
+  - record: seed:kube_pod_container_resource_requests_cpu_cores:sum
+    expr: sum(seed:kube_pod_container_resource_requests_cpu_cores:sum_by_pod)
```

Comment:
I don't think you can create a recording rule based on another recording rule (unless it has already been federated). Has this changed?

Reply:
I was going by the [above](https://github.com/gardener/gardener/pull/1863/files#diff-1751f67b234069990a5af9a44996a6e0R34) existing rule. But then that was already federated?